compatibility/bazel_tools/testing.bzl

```diff
@@ -153,28 +153,30 @@ excluded_test_tool_tests = {
     "0.0.0": [
         {
             "end": "1.0.1-snapshot.20200417.3908.1.722bac90",
-            "exclusions": ["ContractKeysIT"],
```

Comment:
fwiw the fact that CI passes here is because we only run this as part of the daily run. You’ll have to reproduce this locally. If you need help figuring out how to run this (or how to run _only_ the relevant tests) let me know.

Reply:
I'm aware. I've started a local run, but it's taking ages to download the tarballs, so I've also started a full daily run on CI.